docs/eventing/README.md

```diff
@@ -3,6 +3,65 @@ Knative Eventing is a system that is designed to address a common need for cloud
+
+#### Event-Driven Functions
+In the traditional serverless experience, users package their code (the
+functions they want to deploy) and associate them with the event source that
+should start those functions (invoke them). This pattern was made popular by
+hosted Function-as-a-Service platforms like AWS Lambda, Azure Functions, and
+Google Cloud Function.
+
+#### Modern Event Mesh
+As the serverless industry evolved and technology matured, the industry started
+to build architectures where the producers and consumers were even more
+separated. At the same time, the delivery of the events is governed by
+high-level routing and event data is captured in a standard format (like Cloud
+Events). These architectures are facilitated by so called brokers like AWS
+EventBridge and Azure Event Grid.
+
+### Use cases
+
+* When this happens do that (1:1 event delivery): The most straightforward use case is that whenever events are products, you want some code to handle that event. (Read more - coming soon)
+
+* When any of these happen do that (N:1 event delivery): Your functions might be able to handle multiple events. With a standard format for events, like Cloud Events, your function already knows how to handle receiving the event and you’ll only need to update your business logic to handle processing the new data type. (Read more - coming soon)
+
+* When this happens do these (1:N event delivery): Similar to the first scenario, you can have multiple consumers that are all interested in the same event. By leveraging channels we make handling this scenario simple. (Read more - coming soon)
+
+* When this happens do this thing and then another (pipe / tee): When you’re building functionality, it can be useful to split up functionality into smaller chunks. For example, in Unix operating systems the idea is that each command like tool does only one thing. Using the concepts of pipe and tee, you can string these commands together. (Read more - coming soon)
+
+
+* When these things happen do these things and then some (N:M:Z event delivery): So, there’s a lot of things we can do with the constructs introduced in the use cases above, but the one downside with them is that for more complex use cases, where there are a multitude of Sources sending events, Functions consuming/transforming and so forth, it can get a bit unwieldy to keep track of which Channel is handling which events. Furthermore, sometimes you might only want to consume only some events, but with the constructs introduced so far, you would have to receive all of them, and throw out the ones you’re not interested in. To make these kinds of interactions easier and allow the user to only focus on declaring which events they are interested in and where to send them, there is an easier way to reason about them. This is where _Broker_ and _Trigger_ are meant to provide a straightforward user experience. (Read more - coming soon)
```

Comment:
I like the idea of having a better list of use cases on the Eventing landing page, but I think this again is much too wordy and doesn't really explain the different use case in a way that is valuable.